Replaces crayons with markers

BillyBangles opened the pull request:
Exactly what it says on the tin. This PR also adds the custom message system from dddd847 (not a joke this time) as a new type of graffiti, removing a lot of the obvious references to Dark Souls. Also adds a black marker

DISCLAIMER: I wrote the message options with Exodus in mind, not Torch. If any of them are unfitting, they're quite easy to change/remove.

Dinnnertime committed:
Adds message writing to crayon graffiti

Also begins replacing instances of "crayon" with "marker." Sprite
changes to match this coming soon.

Dinnnertime committed:
Replaces crayons with markers

And replacess all instances of "crayon" with "marker" for consistency.
Crayon sprites left in just in case.

Dinnnertime committed:
Adds a black marker.

And the ink/paint for consistency. Changes up the marker box sprite to
fit the new marker.

Dinnnertime committed:
Adds message decal

Replaces the placeholder one I added earlier.

Dinnnertime committed:
Message tweaks

whoops i broke one, dad.

BillyBangles commented:
Here are some screenies for your viewing pleasure.
https://puu.sh/vxqM3/8f1a2da965.png
https://puu.sh/vxq9Y/e2de720c4d.png
https://puu.sh/vxqpy/887c415018.gif

Dinnnertime committed:
Adds changelog

also a message fix (grumble grumble forgot the period grumble)

AtomicWorm commented:
I don't really like this, mostly because the messages are pretty much shitposting.

Besides, if you remove crayons, what are the marines going to eat instead?

BillyBangles commented:
uh

food?

mustafakalash commented:
>because the messages are pretty much shitposting.

That's what the graffiti is for.

Dinnnertime committed:
Lets the marker box actually hold all seven markers

travis please love me

BillyBangles commented:
travis should no longer hate me

update: shit, travis still hates me

Dinnnertime committed:
added more messages

travis senpai notice me please

BillyBangles commented:
at this point i'm not even sure the travis checks are even running at all, without the logs
i don't know what else to do but make minor additions to this until i can get a reliable travis check :V

Dinnnertime committed:
Merge remote-tracking branch 'refs/remotes/Baystation12/dev' into goodworkskeleton

Dinnnertime committed:
Even more messages. Travis pls.

TRAVIS REEEEEEEEEEE

Dinnnertime committed:
More message bugfixes.

jesus fucking christ i need to learn how to code

BillyBangles commented:
FINALLY JESUS
okay i know what the issue is. the crayon box is still mapped into the exodus.
in a perfect world, this next commit should work.

PsiOmegaDelta commented:
Couldn't just have added a marker subtype or similar?

BillyBangles commented:
I could of, but why would SolGov stock their military exploration ship with crayons? As it stands, there aren't even any crayon boxes mapped into the Torch, as far as I can tell.
Similarly, why would NT stock their multi-million thaler research station with crayons? We're not children here.

in other words, ~i hate fun~ it just makes more sense.

Dinnnertime committed:
Replaces crayon box with marker box in the map.

This is the issue I've been having with Travis, but couldn't see because
it wasn't logging any of the damned errors.

mustafakalash commented:
:tada:

BillyBangles commented:
here is your squash
http://www.foodsubs.com/Photos/squash-yellowcrookneck.jpg
for real though is that something i need to do or you guys? I'm a little new to the process here, admittedly.

Dinnnertime committed:
Adds message writing to crayon graffiti

Also begins replacing instances of "crayon" with "marker." Sprite
changes to match this coming soon.

Adds unit test for cryopods setup.

Updates maps to pass the unit test.

Replaces crayons with markers

And replacess all instances of "crayon" with "marker" for consistency.
Crayon sprites left in just in case.

Adds a black marker.

And the ink/paint for consistency. Changes up the marker box sprite to
fit the new marker.

Adds message decal

Replaces the placeholder one I added earlier.

Message tweaks

whoops i broke one, dad.

Adds changelog

also a message fix (grumble grumble forgot the period grumble)

Lets the marker box actually hold all seven markers

travis please love me

Merge pull request #17027 from Datraen/ControllersAndPods

Adds unit test for cryopod controllers
added more messages

travis senpai notice me please

Merge remote-tracking branch 'refs/remotes/Baystation12/dev' into goodworkskeleton

Even more messages. Travis pls.

TRAVIS REEEEEEEEEEE

More message bugfixes.

jesus fucking christ i need to learn how to code

Replaces crayon box with marker box in the map.

This is the issue I've been having with Travis, but couldn't see because
it wasn't logging any of the damned errors.

Dinnnertime committed:
Merge remote-tracking branch 'origin/goodworkskeleton' into goodworkskeleton

BillyBangles commented:
that did it? i think?
or will i need a new PR for that?

AtomicWorm commented:
That... raised the commit count rather than lowering it.

BillyBangles commented:
welp
